src/modules/keyboardq/keyboardq-qt6.qml

```diff
@@ -58,7 +58,7 @@ Item {
         Label {
             id: header
             anchors.horizontalCenter: parent.horizontalCenter
-            text: qsTr("To activate keyboard preview, select a layout.")
+            text: qsTr("Select a layout to activate keyboard preview", "@label")
```

Comment:
Is this a title?

Reply:
No. It appears as an instruction / hint at the top of a window, but it's not a title.

Reply:
![image](https://github.com/calamares/calamares/assets/19431457/9619ce52-394a-493a-a67d-f3f108cbad08)

That one there at the top of the screenshot, partly obscured by the list of physical keyboard layouts.